handler/src/main/java/io/netty/handler/ssl/OpenSslServerSessionContext.java

```diff
@@ -32,36 +31,48 @@ public void setSessionTimeout(int seconds) {
         if (seconds < 0) {
             throw new IllegalArgumentException();
         }
-        SSLContext.setSessionCacheTimeout(context.ctx, seconds);
+        synchronized (context) {
+            SSLContext.setSessionCacheTimeout(context.ctx, seconds);
```

Comment:
`setSessionCacheTimeout` doesn't check for `NULL` and may core dump. We should have a consistent approach for checking for `NULL` between here and the `OpenSslEngine`. If we check for null in java we may avoid crossing the JNI boundary ... but then maybe we should grab the `NULL` value from JNI instead of just checking 0 (as a followup PR).